tests/components/garages_amsterdam/test_config_flow.py

```diff
@@ -0,0 +1,27 @@
+"""Test the Garages Amsterdam config flow."""
```

Comment:
Please exclude the modules that we don't test in `.coveragerc`.

Reply:
Which ones? 😄

Reply:
All except the config flow module.

Reply:
I don't quite understand what you mean, sorry.

Reply:
Like this:
https://github.com/home-assistant/core/blob/f45cd9170370af70425faf8e051f6023b2704334/.coveragerc#L21-L24